depthai_helpers/config_manager.py

```diff
@@ -0,0 +1,342 @@
+import os
+import json
+import platform
+import subprocess
+from pathlib import Path
+import consts.resource_paths
+
+from depthai_helpers.model_downloader import download_model
+from depthai_helpers.cli_utils import cli_print, PrintColors
+
+class DepthConfigManager:
+    labels = ""
+
+    def __init__(self, args):
+        self.args = args
+        self.stream_list = args['streams']
+        self.compile_model = self.args['shaves'] is not None and self.args['cmx_slices'] is not None and self.args['NN_engines']
```

Comment:
should be `or` instead of `and` (as if I change just `NN_engines` the model should be also compiled

Reply:
That makes sense, I suppose. I just moved the existing logic which requires all three to be passed in order to have that compile variable set. I can change it to do that though. Just make it default to
            shave_nr = 7
            cmx_slices = 7
            NCE_nr = 1
if nothing is passed for one of the three?

Reply:
yes, good call!